project.clj

```diff
@@ -1,10 +1,10 @@
 (defproject nomnom/bunnicula.monitoring "2.1.0"
   :description "Monitoring middleware for Bunnicula, with built in support for StatsD metrics and error reporting to Rollbar"
   :url "https://github.com/nomnom-insights/nomnom.bunnicula.monitoring"
-  :dependencies [[org.clojure/clojure "1.10.1"]
-                 [com.stuartsierra/component "0.4.0"]
-                 [org.clojure/tools.logging "0.5.0"]
-                 [nomnom/bunnicula "2.1.0"]
+  :dependencies [[org.clojure/clojure "1.10.3"]
+                 [com.stuartsierra/component "1.0.0"]
+                 [org.clojure/tools.logging "1.1.0"]
+                 [nomnom/bunnicula "2.2.2"]
```

Comment:
Don't forget this will need to be updated once you publish new nomnom/bunnicula version.

Reply:
I didn't change previous bunnicula version number, there are no changes on code. This is tricky, should we?

Reply:
If we updated dependencies we need to release it as new version (as underlying code changed).